homeassistant/components/xiaomi_aqara/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "Xiaomi Gateway (Aqara)",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/xiaomi_aqara",
-  "requirements": ["PyXiaomiGateway==0.14.1"],
+  "requirements": ["PyXiaomiGateway==0.14.3"],
```

Comment:
Side note: The `bind_interface` parameter isn't used anymore:
https://github.com/Danielhiversen/PyXiaomiGateway/blob/094f714de58ceab42329b1ea7f780c1af878669b/xiaomi_gateway/__init__.py#L23